### ISSUE TITLE
Example code fails as "No such file or directory" if working-directory is modified

### ISSUE THREAD

**Issue by yuku** [google-github-actions/auth#40]
### TL;DR

[README > Examples > Cloud SDK (gcloud) @v0.3.1](https://github.com/google-github-actions/auth/tree/v0.3.1#cloud-sdk-gcloud) fails if `working-directory` is modified:

![image](https://user-images.githubusercontent.com/96157/140834761-3f1b98ba-4d59-47ed-bf09-3d41731a5cb0.png)

Workaround:

```yaml
defaults:
  run:
    working-directory: somewhere
# ...
jobs:
  job_id:
    steps:
      # ...
      - run: gcloud auth login --brief --cred-file="${{ steps.auth.outputs.credentials_file_path }}"
        working-directory: .  # Add this line. Use the default working-directory
```

**Expected behavior**

This is not a bug of this repository. Worth mention in README?

**Observed behavior**

### Reproduction

**Action YAML**

```yaml
defaults:
  run:
    working-directory: somewhere
jobs:
  job_id:
    runs-on: ubuntu-latest
    permissions:
      id-token: write
    steps:
      - uses: 'google-github-actions/setup-gcloud@master'
        with:
          project_id: my-project-id
      - uses: google-github-actions/auth@v0.3.1
        id: auth
        with:
          create_credentials_file: 'true'
          workload_identity_provider: projects/1234567890/locations/global/workloadIdentityPools/pool-id/providers/provider-id
          service_account: my-service-account@my-project-id.iam.gserviceaccount.com
      - run: gcloud auth login --brief --cred-file="${{ steps.auth.outputs.credentials_file_path }}"
```

**Repository**

**Additional information**

I think this is not a bug. I'm just writing this issue to save someone's time in the future.

I'm wondering why this happens even the `steps.auth.outputs.credentials_file_path` returns an absolute path... 🤔
